### PR TITLE
Feature/pfappserver nodes update

### PR DESCRIPTION
**Pull request by satkunas**
# Description
Continue work on the pfappserver nodes

# Impacts
Increased coffee intake

# Delete branch after merge
YES

### CODE REVIEW

File: html/pfappserver/root/static.alt/src/components/pfSearch.vue [hunk 1]
```diff
@@ -1,49 +1,42 @@
...
+      <pf-search-boolean :model="condition" :fields="fields" :store="store" :advancedMode="advancedMode"/>
+      <br/>
+      <b-container fluid class="mt-3 px-0 text-right">
+        <b-button type="reset" variant="outline-secondary">{{ $t('Reset') }}</b-button>
+        <b-button type="submit" variant="outline-primary">{{ $t('Search') }}</b-button>
+      </b-container>
+      </b-form>
     </div>
+    <b-form @submit.prevent="onSubmit" @reset.prevent="onReset" v-else>
+    <div class="input-group">
```
Comment (cgx) [inline html/pfappserver/root/static.alt/src/components/pfSearch.vue]: Please fix the indentation of this `<div>` block.

File: html/pfappserver/root/static.alt/src/components/pfSearchBoolean.vue [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+        this.model.values[outerindex].values.splice(innerindex, 1)
+      }
+    }
+  },
+  mounted () {
+  }
+}
+</script>
+
+<style lang="scss" scoped>
```
Comment (cgx) [inline html/pfappserver/root/static.alt/src/components/pfSearchBoolean.vue]: `rc-b, rc-br, rc-l, rc-tl, rc-tr` don't seem to be used.

File: html/pfappserver/root/static.alt/src/views/Nodes/_components/NodesSearch.vue [hunk 3]
```diff
@@ -528,9 +553,11 @@ export default {
   watch: {
     checkedRows (a, b) {
       this.checkedAll = (this.tableValues.length === a.length && a.length > 0)
+      let _this = this
       let checkedRows = this.checkedRows
       this.items.forEach(function (item, index, items) {
-        items[index]._rowVariant = checkedRows.includes(item) ? 'info' : ''
+        // items[index]._rowVariant = checkedRows.includes(item) ? 'info' : ''
```
Comment (cgx) [inline html/pfappserver/root/static.alt/src/views/Nodes/_components/NodesSearch.vue]: Remove this line.

**Review by cgx — CHANGES_REQUESTED** [review]

File: html/pfappserver/root/static.alt/src/views/Nodes/_components/NodesSearch.vue [hunk 4]
```diff
@@ -489,13 +509,45 @@ export default {
...
-      console.log(['applyDeregister', this.checkedRows])
+      const _this = this
+      const macs = []
+      this.checkedRows.forEach(function (item, index, items) {
+        macs.push(item.mac)
+        // _this.$store.dispatch('$_nodes/deregisterNode', item.mac).then(response => {
+        //   _this.$store.commit('$_nodes/NODE_VARIANT', {mac: item.mac, variant: 'success'})
+        // }).catch(() => {
+        //   _this.$store.commit('$_nodes/NODE_VARIANT', {mac: item.mac, variant: 'danger'})
+        // })
```
Comment (cgx) [inline html/pfappserver/root/static.alt/src/views/Nodes/_components/NodesSearch.vue]: Can those commented lines be removed?

File: html/pfappserver/root/static.alt/src/views/Nodes/_components/NodesSearch.vue [hunk 5]
```diff
@@ -428,11 +427,28 @@ export default {
...
     },
     onReset () {
       this.requestPage = 1 // reset to the first page
       this.$store.dispatch('$_nodes/setSearchQuery', undefined) // reset search
       this.$store.dispatch('$_nodes/search', this.requestPage)
+      this.clearChecked()
+      this.initSearch()
+    },
+    initSearch () {
+      if (this.$route.query.query) {
```
Comment (cgx) [inline html/pfappserver/root/static.alt/src/views/Nodes/_components/NodesSearch.vue]: I guess this should be exposed as a prop and access through `this.query`.

**Review by cgx — APPROVED** [review]

**Comment by satkunas** [comment]
Attempting to re-open w/ new branch

**Pull request merged**